api/src/main/java/openconsensus/trace/Span.java

```diff
@@ -117,6 +117,16 @@
    */
   public abstract void setStatus(Status status);
 
+  /**
+   * Updates the {@code Span} name.
+   *
+   * <p>If used, this will override the name provided via {@code SpanBuilder}.
```

Comment:
add a comment that sampling will not be re-evaluated

Reply:
more accurately, the sampling behavior depends on the implementation.